com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.pyi

```diff
@@ -31,6 +31,7 @@ class Topology(object):
     def add_file_dependency(self, path: str, location: str) -> str: ...
     def add_pip_package(self, requirement: str) -> None: ...
     def create_submission_parameter(self, name:str, default=None, type_=None): ...
+    def checkpoint_period(self, period: int, uints: TimeUnit) -> None: ...
```

Comment:
Signature is wrong, needs to be a union of float & datetime.timedelta

Reply:
I forgot to update that.  What do you mean by a union of float and datetime.timedelta?  Is there some support for c-like union data types in python?